docs/resources/rds_read_replica_instance.md

```diff
@@ -47,6 +47,10 @@ resource "huaweicloud_rds_read_replica_instance" "replica_instance" {
   primary_instance_id   = huaweicloud_rds_instance.instance.id
   availability_zone     = "{{ availability_zone }}"
   enterprise_project_id = "{{ enterprise_project_id }}"
+  charging_mode         = "prePaid"
+  period_unit           = "month"
+  period                = 1
+  auto_renew            = "true"
```

Comment:
generally, the prepaid charging mode is not shown in basic usage.

Reply:
done